docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,168 @@
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/test-replay-header.jpg"
+  alt="Test Replay header"
+/>
+
+### Command Log
+
+The left side of the screen displays the [command log](/guides/core-concepts/cypress-app#Command-Log) just like in the Cypress app, where you can step through, inspect and [time travel](/guides/core-concepts/cypress-app#Time-traveling) debug your test.
+
+<DocsImage
+  src="/img/guides/core-concepts/cypress-app/command-log.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Developer Tools
+
+Dive into network-level traffic, console events and inspect the application under test just as you can in the browser.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/dev-tools.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Replay Controls
+
+Toggle each test attempt, play back errors at a variety of speeds or scrub through the captured error by dragging the playhead along the timeline.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/scrubber.png"
+  alt="Test Replay scrubber controls"
+/>
+
+## Opt out of Test Replay
+
+:::warning
+
+<Icon name="exclamation-triangle" /> Do not use sensitive data in your tests. This
+will be visible in the test replay.
+
+:::
+
+If you do not want to capture test data for replay and debugging purposes simply "shut off" Test Replay in project settings.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/opt-out.png"
+  alt="Test Replay opt-out controls"
+/>
+
+## Troubleshooting
+
+**If Test Replay is not present or disabled in Cypress Cloud:**
+
+1. Confirm runs have been recorded to Cypress Cloud using Cypress `v13` or later.
+2. Ensure tests were recorded using a Chromium-based browser(Chrome, Edge, Electron).
+3. Check that Test Replay is enabled in Cypress Cloud project settings.
+
+## Command Log Notes
+
+Cypress uses a powerful [command log](/guides/core-concepts/cypress-app#Command-Log) locally to display every command executed in your spec. The Cypress command log is visible during `cypress open` mode, but is also rendered ([headless](/guides/guides/command-line#Options)-ly) during `cypress run` so that it displays within captured screenshots or videos, to assist in debugging CI failures.
+
+### Problem
+
+Rendering the Cypress command log while running tests via `cypress run` often has performance implications, resulting in a slower test runtime, especially on lower resourced machines.
+
+### Solution
+
+With Test Replay, Cypress regenerates the entire UI within the Cloud for viewing the run and the debugging CI failures. Additionally, displaying the Cypress command log in videos and screenshots is less valuable for users using Test Replay.
+
+Because of this, as of Cyrpess `v13`, the Cypress command log is no longer rendered during `cypress run` for users using Test Replay and `video: false`, with an option to opt out of this behavior.
```

Comment:
Have we made the decision to no longer show the reporter by default? This was up in the air last time I checked and not sure if a decision was made.

Reply:
Sounds like the decision to hide it is made. we might want to reference the new CLI options here in case users want to use them (original comment https://github.com/cypress-io/cypress-documentation/pull/5349#discussion_r1297296350)